doc/ug/io.tex

```diff
@@ -1217,6 +1217,256 @@ \section{Error handling}
 continue, so you should really know what you do if you try and catch
 these errors.
 
+\section{Online-visualisation with Mayavi or OpenGL}
+\label{sec:visualizer}
+\index{Visualization}
+
+With the python interface, \es features two possibilities for
+online-visualization: 
+
+\begin{enumerate}
+\item Using the mlab module to drive \emph{Mayavi, a "3D scientific data
+visualization and plotting in Python"}. Mayavi has a user-friendly GUI to
+specify the appearance of the output. Additional requirements: XXXXXX.
```

Comment:
Left that open for @mkuron but forgot to discuss..
Also just saw that there's a blank page for 'latex' reasons and the pysyntax environment doesn't seem to support argument lists spanning over more than one page. @hmenke?